libs/commons/Logs_.ml

```diff
@@ -40,9 +40,18 @@ let default_skip_libs =
     "cohttp.lwt.client";
     "cohttp.lwt.io";
     "conduit_lwt_server";
-    "mirage-crypto-rng.lwt";
-    "mirage-crypto-rng-lwt";
-    "mirage-crypto-rng.unix";
+    "dns*";
```

Comment:
add a comment above before default_skip_libs explaining you can use regexps.
Ideally you could even do let default_skip_libs : Pcre.regexp t = [
 ] |> List.map Pcre.regexp